styles.css

```diff
@@ -0,0 +1,575 @@
+}
+
+.image2-hr {
+  width: 50px;
+}
+
+.image3Cont {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.smallPar {
+  font-size: 12px;
+  color: grey;
+  display: block;
+  text-align: left;
+  margin-left: 145px;
+}
+
+.lastPar {
+  margin-left: 150px;
+  margin-right: 150px;
+  font-size: 12px;
+  color: grey;
+  justify-content: center;
+}
+
+.lastparAnchor {
+  font-size: 12px;
+  color: grey;
+  text-decoration: underline;
+  margin-right: 5px;
+  margin-left: 5px;
+}
+
+.lastparAnchor:hover {
+  text-decoration: none;
+}
+
+.commentCon {
+  margin: 30px 150px 30px 150px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.comments-button {
+  grid-column: 1 / 4;
+  border: #326891 1px solid;
+  background-color: #567b95;
+  color: white;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  text-decoration: none;
+}
+
+.comments-button:hover {
+  background-color: #326891;
+}
+
+/* RECOMENDED ARTICLES */
+.footerWeb {
+  justify-content: center;
+  display: grid;
+  grid-template-columns: 778px 333px;
+  grid-template-rows: 400px;
+  height: 900px;
+}
+
+.footCon1 {
+  border-top: 1px solid black;
+  display: grid;
+  grid-template-columns: 259px 259px 259px;
+  font-size: 15px;
+}
+
+.articles {
+  margin-top: 40px;
+}
+
+.articles img {
+  width: 220px;
+}
+
+.border-top {
+  margin-top: 20px;
+  border-top: 1px solid black;
+  display: flex;
+  width: 777px;
+}
+
+.border-top .articles {
+  margin-right: 40px;
+}
+
+.footCon2 {
+  margin-left: 20px;
+  border-top: 1px solid black;
+}
+
+.articleSource {
+  text-align: right;
+  color: grey;
+  font-size: 13px;
+}
+
+.articleDescription {
+  text-align: left;
+  color: black;
+  font-size: 17px;
+  font-weight: lighter;
+}
+
+.articleDate {
+  text-align: left;
+  color: grey;
+  font-size: 13px;
+}
+
+/* MOST POPULAR */
+.aside2 {
+  font-size: 15px;
+}
+
+.most-popular-header {
+  position: absolute;
+  margin-top: 10px;
+}
+
+.aside2 li:first-of-type {
+  list-style: none;
+  text-align: left;
+  color: black;
+  padding: 10px 0;
+}
+
+.aside2 li:not(:first-of-type) {
+  list-style: none;
+  text-align: left;
+  color: black;
+  border-top: 1px solid grey;
+  padding: 10px 0;
+}
+
+.aside2Header {
+  font-family: 'Arial', sans-serif;
+  font-weight: bolder;
+}
+
+.lists-out-container {
+  display: flex;
+  margin-bottom: 50px;
+  justify-content: center;
+  margin-left: 40px;
+}
+
+.lists-container {
+  display: flex;
+  width: 1210px;
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+  justify-content: space-between;
+  line-height: 20px;
+}
+
+.final-container {
+  width: 1210px;
+}
+
+.subscription-container {
+  width: 260px;
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+  padding-left: 20px;
+  line-height: 20px;
+  border-left: 1px solid grey;
+}
+
+.lists-container ul {
+  margin-right: 40px;
+}
```

Comment:
This selector and selector on line 433 have the same style definition. Group selectors with the same style definition.